js/src/components/contact-information/phone-number-card.scss

```diff
@@ -0,0 +1,42 @@
+.gla-phone-number-card {
+	// Country code selector
+	.wcdl-select-control {
+		.wcdl-select-control__input {
+			margin-bottom: 0;
+		}
+
+		.woocommerce-select-control {
+			.components-base-control {
+				height: 36px;
+				border-color: $gray-600;
+			}
+
+			.woocommerce-select-control__control-input {
+				font-size: 13px;
+				color: $gray-900;
+			}
+
+			.woocommerce-select-control__listbox {
+				top: 37px;
+			}
+
+			.woocommerce-select-control__option {
+				min-height: 36px;
+				font-size: 13px;
```

Comment:
I was thinking of using some variables shared with `woocommerce/components` here, to keep it consistent and futureproof, but it seems `13` and `36` are hardcoded there as well :|